Source/Core/Core/HW/WiimoteEmu/ExtensionPort.h

```diff
@@ -22,6 +22,8 @@ enum ExtensionNumber : u8
   UDRAW_TABLET,
   DRAWSOME_TABLET,
   TATACON,
+
+  BALANCE_BOARD,
```

Comment:
The  empty lines in this list are unnecessary imo.

Reply:
I think it's fine either way.

Reply:
I added the blank line because the balance board isn't a typical extension and doesn't directly fit within the rest of the list.  IMO I'd rather not remove the blank line (except if `BALANCE_BOARD` is removed from the whole `ExtensionNumber` enum, which would require other refactoring but ideally it shouldn't be selectable for other controllers, which I don't think I implemented yet).